src/gui/log/loglistview.h

```diff
@@ -1,6 +1,7 @@
 /*
  * Bittorrent Client using Qt and libtorrent.
- * Copyright (C) 2011  Christophe Dumez <chris@qbittorrent.org>
+ * Copyright (C) 2019  sledgehammer999 <hammered999@gmail.com>
+ * Copyright (C) 2020  Prince Gupta    <jagannatharjun11@gmail.com>
```

Comment:
later date should be on top.

Reply:
And don't appent these extra spaces inside this line (between name and email).

Reply:
done